fuel/app/themes/default/partials/my_widgets.php

```diff
@@ -321,7 +321,7 @@ class="action_button arrow_down_button">
 				</div>
 				<div class="scores">
 					<h2>Student Activity</h2>
-					<span id="export_scores_button" class="action_button aux_button" ng-click="exportPopup()">
+					<span id="export_scores_button" class="action_button aux_button {{selected.scores.list.length === NULL ? 'disabled' : ''}}" ng-click="selected.scores.list.length === NULL ? angular.noop() : exportPopup()">
```

Comment:
This is a case where https://docs.angularjs.org/api/ng/directive/ngClass would be a bit cleaner to read and wouldn't potentially interfere with the regular `class` application.
```
ng-class="{'disabled': selected.scores.list.length === NULL}"
```